ckanext/canada/plugin/public_plugin.py

```diff
@@ -212,6 +215,56 @@ def column_types(self, existing_types: Dict[str, Type[ColumnType]]):
     def get_blueprint(self) -> List[Blueprint]:
         return [canada_views]
 
+    def _canada_citation_map(self, cite_data: DataDict,
+                             pkg_dict: DataDict) -> Dict[str, Any]:
+        lang = 'en'
+        if has_request_context():
+            lang = h.lang()
```

Comment:
does `h.lang()` raise an exception when there's no request context? If so it's considered better style to do this with a `try:` `except:`